src/subcommands/get.go

```diff
@@ -31,7 +35,15 @@ func SubcommandGet(g *pb.GetRequest, indexPath string, w io.Writer) (string, *pb
 				fmt.Fprintf(w, "failed to read content from copy: %v. Fetching from the source.\n", err)
 			}
 
-			content, err = fetch.FetchFromSource(targetEntry.ContentMetadata.SourceType, targetEntry.ContentMetadata.URI)
+			targetMetadata, err := db.GetContentMetadata(ctx, conn, g.Name)
```

Comment:
We've already accessed the content Metadata from the index at this point, why read both? Should we maybe have a CLI/proto arg that controls which we read from (at this point. In the future we probably do want to refactor/design things so that we only read from one)?

Reply:
Yes, we don't need to read from the index list in the future. Just the database is enough. I have kept that part as legacy code which needs to be refactored later. We can add a proto arg for now to choose from which index to read (file or db). But even that is redundant as we mandate a db connection right now, which makes the file just a duplicate of the db